ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 50, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 50, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMBAT010' has no attribute 'KCOMBAT010'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 45, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 45, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    mod = importlib.import_module(basic_ui_route + '.' + menu[0].pgm_id)
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    mod = importlib.import_module(basic_ui_route + '.' + menu[0].pgm_id)
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    mod = importlib.import_module(basic_ui_route + '.' + menu[0].pgm_id)
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 45, in loadUI
    uiClass = getattr(mod, menu[0].pgm_id)()
AttributeError: module 'UI._uiFiles.COM.KCOMDEV010' has no attribute 'KCOMDEV010'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 44, in loadUI
    mod = importlib.import_module(basic_ui_route + '.' + menu[0].pgm_id)
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 45, in loadUI
    mod = importlib.import_module(basic_ui_route + '.' + menu[0].pgm_id)
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 45, in loadUI
    mod = importlib.import_module(basic_ui_route + '.' + menu[0].pgm_id)
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 32, in __init__
    self.initUI()
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 37, in initUI
    self.test()
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 80, in test
    mod = importlib.import_module(basic_ui_route + '.' + 'KCOMMAN002')
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 81, in test
    mod = importlib.import_module(basic_ui_route + '.' + 'KCOMMAN002')
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 81, in test
    mod = importlib.import_module(basic_ui_route + '.' + 'KCOMMAN002')
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 82, in test
    mod = importlib.import_module(basic_ui_route + '.' + 'KCOMMAN002')
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 45, in loadUI
    mod = importlib.import_module(basic_ui_route + '.' + menu[0].pgm_id)
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 82, in test
    mod = importlib.import_module(basic_ui_route + '.' + 'KCOMMAN002')
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'UI._uiFiles.COM.KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 82, in test
    mod = importlib.import_module('KCOMMAN002')
  File "C:\ProgramData\Anaconda3\envs\rep\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'KCOMMAN002'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 83, in test
    uiClass = getattr(mod, 'KCOMMAN020')()
AttributeError: module 'KCOMMAN020' has no attribute 'KCOMMAN020'

ERROR:root:Traceback (most recent call last):
  File "C:/Users/Ceasar.DESKTOP-AQTREV4/PycharmProjects/rep/61.WorkSpace/Almighty/UI/_uiFiles/COM/KCOMMAN010.py", line 83, in test
    uiClass = getattr(mod, 'KCOMMAN020')()
AttributeError: module 'UI._uiFiles.COM.KCOMMAN020' has no attribute 'KCOMMAN020'

